frontend/src/lib/candidate/components/textArea/TextArea.type.ts

```diff
@@ -33,4 +33,33 @@ export type TextAreaProps = {
    * @default false
    */
   disabled?: boolean;
+  /**
+   * Background color of the textarea.
+   * @default 'bg-bases-100'
+   */
+  bgColor?: string;
+  /**
+   * The placeholder text for the textarea.
+   * @default undefined
+   */
+  placeholder?: string;
+};
+
+export type MultilangTextAreaProps = {
+  /**
+   * The text content of the text area.
+   */
+  multilangText: LocalizedString;
+  /**
+   * Previously saved text content from the database.
+   * Is initially shown if nothing is in local storage.
+   * @default undefined
+   */
+  previouslySavedMultilang?: LocalizedString | undefined | null;
```

Comment:
Same thing here: when the question mark is given after the variable name, the undefined union type can be omitted.

Why do we allow a null value here? If there is some reason, I think it should be documented in the inline docs above for clarity.